packages/compass-home/src/components/workspace-content.spec.tsx

```diff
@@ -20,28 +20,18 @@ const getComponent = (name: string) => {
 };
 
 describe('WorkspaceContent [Component]', function () {
-  let testAppRegistry: AppRegistry;
-  beforeEach(function () {
-    testAppRegistry = new AppRegistry();
```

Comment:
@addaleax for now swithcing it back to global app registry in the `useAppRegistryComponent` and tests, the usage of those will be removed anyway pretty soon as the plugins we mock here are the next in line to be refactored.

Now that we have an app registry scope for a connected part application, we can't rely on global === local in this part of the app to ease testing